src/Tablebot/Plugins/Reminder.hs

```diff
@@ -124,7 +124,7 @@ reminderCron = do
             Left _ -> pure ()
```

Comment:
This isn't caused by your PR (it was a bug from before) - but if a reminder command's original message (the starting `.remind ...` message) is deleted, the reminder will never be reminded about and will be brought up by the database with every run of the cron job without actually doing the reminder.